src/Pate/Verification/StrongestPosts.hs

```diff
@@ -157,11 +157,57 @@ pairGraphComputeFixpoint gr =
   case chooseWorkItem gr of
     Nothing -> return gr
     Just (gr', nd, preSpec) -> do
-      gr'' <- PS.viewSpec preSpec $ \vars asm d -> do
-        withAssumptionSet asm $
-          visitNode nd vars d gr'
+      gr'' <- PS.viewSpec preSpec $ \scope d -> do
+        withAssumptionSet (PS.scopeAsm scope) $
+          visitNode scope nd d gr'
       pairGraphComputeFixpoint gr''
 
+
+
+absValueToAsm ::
```

Comment:
How does the `AbsValue` turn into an `AssumptionSet`? It looks like, at a high level, this is doing the offset remapping step. Probably worth a few words here, or at least a pointer to the overall description of the tricky bits here

Reply:
Added a comment